### ISSUE TITLE
Use local flow-bin package on OSX

### ISSUE THREAD

**Issue by ssenchenko** [flowtype/flow-for-vscode#240]
I know you had similar issue (#69) and closed it. However, I have the same problem on OSX.
My configuration:
* flowtype version 0.8.1 (recently installed)
* "flow.useNPMPackagedFlow": true,
* "flow.pathToFlow": "${workspaceFolder}/client/web/node_modules/.bin/flow"

I keep on getting the error from VSCode:
[Flow] Cannot find flow in PATH. Try to install it by npm install flow-bin -g

I don't think that it's a good idea to install flow globally or add project specific node_modules/.bin/flow to PATH.

**Comment by orta** [comment]
This feels like a bug  with getting the flow binary to me

**Comment by budarin** [comment]
I have the error on Windows too.
It only accepts full path which is not sutable for any :(

**Comment by KeitIG** [comment]
I have the same problem, here's my config:

```
"javascript.validate.enable": false,
"flow.useNPMPackagedFlow": true,
"flow.pathToFlow": "${rootPath}/website/node_modules/.bin/flow",
```

**Comment by aligfl** [comment]
Seeing the same issue. even though i have i have the path set:
```"flow.pathToFlow": "${workspaceFolder}/node_modules/flow-bin/flow-osx-v0.81.0/flow",```

I'm still getting:
```[Flow] Cannot find flow in PATH. Try to install it by npm install flow-bin -g```

**Comment by Mayank1791989** [comment]
Fixed in v1.1.0
